### PR TITLE
User can prevent posts from keep updating issue #163

### PR DESCRIPTION
**Pull request by mitjap**
#163

### CODE REVIEW

**Comment by mitar** [comment]
Please update from main repository. It cannot be automatically merged.

File: piplmesh/frontend/static/piplmesh/js/home.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 var POSTS_LIMIT = 20;
 var POSTS_DATE_UPDATE_INTERVAL = 60000; // ms
 
+// is new post automatically posted
+var CHANNEL_QUEUE_ENABLED = [];
```
Comment (mitar) [inline piplmesh/frontend/static/piplmesh/js/home.js:5]: This is not constant. Also, global variables should be kept to minimum.

**Comment by mitar** [comment]
This is very complicated code. You could simply always add new posts to the list, only hide them. Because there are in general problems of possibility of duplicate posts. So it is much easier if you always add posts to the list, just that you do not display them if they are hidden. And when user clicks, you just display them.

**Comment by mitar** [comment]
Please update/merge from main repository.

**Comment by mitar** [comment]
This pull request is garbage. You will have to do it again.

**Comment by mitar** [comment]
(Not the content, but how you merged.)

**Comment by mitjap** [comment]
I totally agree :)

**Pull request closed without merging**